src/picknik_ur_base_config/objectives/move_to_joint_state.xml

```diff
@@ -3,7 +3,7 @@
     <BehaviorTree ID="Move to Joint State" _description="Single-node behavior to move to a specified joint state">
         <Control ID="Sequence">
             <Action ID="GetStringFromUser" parameter_name="move_to_joint_state.waypoint_name" parameter_value="{waypoint_name}" />
-            <Action ID="MoveToJointState" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" planning_group_name="manipulator" waypoint_name="{waypoint_name}" />
+            <Action ID="MoveToJointState" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" planning_group_name="manipulator" waypoint_name="{waypoint_name}"/>
```

Comment:
> @eholum The parallel error is something I ran into sporadically as well -- am talking to @sjahr about it to see what it could be.

The warning is produced [here](https://github.com/PickNikRobotics/moveit_studio/blob/18f59a926de03609106e6b5e53f67d63e20e5c6c/src/moveit_studio_behavior/src/utils/mtc_utils.cpp#L86C6-L86C35). If the parallel planning port is not set, we default to not using parallel planning at all.

To fix this you need to update your objective files like this:
```suggestion
            <Action ID="MoveToJointState" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" planning_group_name="manipulator" waypoint_name="{waypoint_name}" use_all_planners="true"/>
```
If you set it to false, no parallel planning is used (not a bad thing) but the warning disappears too.

Reply:
@kineticsystem You need to apply this to all objectives that use `MoveToJointState`

Reply:
Oh, so this was a red herring! The warning happens regardless, but the errors are because planning failed! Interesting...